cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
Good job using the `validate` method in your `TicketSerializer` to ensure that the ticket is valid before it is created. This is a great use of Django's built-in validation.